sources/AppBundle/Event/Model/Talk.php

```diff
@@ -258,4 +278,148 @@ public function setNeedsMentoring($needsMentoring)
 
         return $this;
     }
+
+    /**
+     * @return int
+     */
+    public function getYoutubeId()
+    {
+        return $this->youTubeId;
+    }
+
+    /**
+     * @param int $youtubeId
+     *
+     * @return Talk
+     */
+    public function setYoutubeId($youtubeId)
+    {
+        $this->propertyChanged('youtubeId', $this->youTubeId, $youtubeId);
+        $this->youTubeId = $youtubeId;
+
+        return $this;
+    }
+
+    /**
+     * @return null|string
+     */
+    public function hasYoutubeId()
+    {
+        return null !== $this->youTubeId;
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getYoutubeUrl()
+    {
+        if (!$this->hasYoutubeId()) {
+            return null;
+        }
+
+        return 'https://www.youtube.com/watch?v=' . $this->getYoutubeId();
+    }
+
+    /**
+     * @return int
+     */
+    public function getSlidesUrl()
+    {
+        return $this->slidesUrl;
+    }
+
+    /**
+     * @param int $slidesUrl
+     *
+     * @return Talk
+     */
+    public function setSlidesUrl($slidesUrl)
+    {
+        $this->propertyChanged('slidesUrl', $this->slidesUrl, $slidesUrl);
+        $this->slidesUrl = $slidesUrl;
+
+        return $this;
+    }
+
+    /**
+     * @return bool
+     */
+    public function hasSlidesUrl()
+    {
+        return null !== $this->slidesUrl;
+    }
+
+    /**
+     * @return int
+     */
+    public function getJoindinId()
+    {
+        return $this->joindinId;
+    }
+
+    /**
+     * @param int $joindInId
+     *
+     * @return Talk
+     */
+    public function setJoindinId($joindInId)
+    {
+        $this->propertyChanged('joindinId', $this->joindinId, $joindInId);
+        $this->joindinId = $joindInId;
+
+        return $this;
+    }
+
+    /**
+     * @return bool
+     */
+    public function hasJoindinId()
+    {
+        return null !== $this->joindinId;
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getJoindinUrl()
+    {
+        if (!$this->hasJoindinId()) {
+            return null;
+        }
+
+        return 'https://legacy.joind.in/talk/view/' . $this->getJoindinId();
```

Comment:
les ids sont différents sur la nouvelle version : https://joind.in/talk/5117a je n'ai vu comment linker sur la nouvelle version avec les anciens ids.